Level_2/string_compression.py

```diff
@@ -0,0 +1,26 @@
+def solution(s: str) -> int:
+    if len(s) == 1:
+        return 1
+    answer = []
+    for i in range(1, (len(s)//2) + 1):
```

Comment:
ㅋㅋ나도 여기서 틀렸는뎈ㅋㅋㅋㅋㅋ
요런 방법도 있음!
```suggestion
    answer = []
    for i in range(1, (len(s)//2) + 2):
```